src/server/server.ts

```diff
@@ -128,6 +147,47 @@ async function serveArticle(req: Request, res: ExpressResponse): Promise<void> {
     }
 }
 
+const liveBlockUpdates = (since: number, content: Content): LiveUpdates => ({
+    newBlocks: serialiseLiveBlocks(newBlocksSince(since)(content)(docParser)),
+    updatedBlocks: serialiseLiveBlocks(updatedBlocksSince(since)(content)(docParser)),
+});
+
+const recentLiveBlocks = (content: Content): LiveUpdates => ({
+    newBlocks: serialiseLiveBlocks(recentBlocks(10)(content)(docParser)),
```

Comment:
Could this be something we could configure?
`const blocksPerRequest = 10`

Reply:
Yup, see no reason why not, that's why I included it as a param rather than hardcoding it. This is really just a fallback for the moment to make the API well-formed (i.e. what happens when you hit the endpoint with no query param), I don't know if anything on the client will actually hit this.